frontend/src/pages/TermPlanner/TermBox/styles.js

```diff
@@ -45,6 +45,7 @@ const TermCheckboxWrapper = styled.div`
   justify-content: center;
   padding-top: 3.5px;
   transition: all 200ms ease;
+  color: ${({ checked }) => (checked ? "#000" : "#000")}; 
```

Comment:
do we need a ternary since its returning the same value (or even the use of the `checked` prop?